src/multi-select-item/multi-select-item.component.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
```

Comment:
The multi-select-item folder and contents could be moved under multi-select since it is a sub-component.